jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/tasklogs/TaskLogItemView.java

```diff
@@ -143,13 +95,29 @@ public void setValue(final TaskEventSummary model) {
                 iconClass += " fa fa-cogs";
             }
         }
-        tooltip(logIcon);
-        String logString = constants.Task() + " " + translationService.format(type.getTypeTranslationId()).toLowerCase();
-        logIcon.setAttribute("data-original-title",
-                             logString);
-        logIcon.setClassName(iconClass);
-        logTypeDesc.setTextContent(logString);
+        return iconClass;
+    }
 
+    private void setLogInfo(final TaskEventType type, final TaskEventSummary model) {
+        String logInfoContent = constants.ByUser() + " " + model.getUserId() + " ";
+        switch (type) {
+            case ADDED: {
+                if (model.getUserId() != null && !model.getUserId().isEmpty()) {
+                    logInfoContent = constants.ByProcess() + " '" + model.getUserId() + "' ";
```

Comment:
Do we need to end the string with whitespace after an apostrophe?

Reply:
@jomarko I think this is question for @nmirasch as I only refactored her code :). But I think it is because this string is followed by another string in the log (and it cannot be in one string because they have different style).

Reply:
yes it's true, in this case this white space is not needed, I was included when some other log info was concatenated and to ensure they were separated, It can be removed